noxfile.py

```diff
@@ -57,62 +51,55 @@ def run_integration(session):
 
 @nox.session(name="doctests", reuse_venv=True)
 def run_doctests(session):
+    set_environment_variables(session=session)
     session.install("-e", ".[docs]")
     session.run("python", "run-tests.py", "--doctest")
 
 
 @nox.session(name="unit", reuse_venv=True)
 def run_unit(session):
-    homedir = os.getenv("HOME")
-    session.env["SUNDIALS_INST"] = session.env.get("SUNDIALS_INST", f"{homedir}/.local")
-    session.env[
-        "LD_LIBRARY_PATH"
-    ] = f"{homedir}/.local/lib:{session.env.get('LD_LIBRARY_PATH')}"
+    set_environment_variables(session=session)
     session.install("-e", ".")
     if sys.platform == "linux":
-        session.run("pybamm_install_jax")
         session.install("scikits.odes")
+        session.run("pybamm_install_jax")
     session.run("python", "run-tests.py", "--unit")
 
 
 @nox.session(name="examples", reuse_venv=True)
 def run_examples(session):
+    set_environment_variables(session=session)
     session.install("-e", ".[dev]")
     session.run("python", "run-tests.py", "--examples")
 
 
 @nox.session(name="dev", reuse_venv=True)
 def set_dev(session):
-    homedir = os.getenv("HOME")
-    LD_LIBRARY_PATH = f"{homedir}/.local/lib:{session.env.get('LD_LIBRARY_PATH')}"
+    set_environment_variables(session=session)
     envbindir = session.bin
-    session.install("-e", ".[dev]")
```

Comment:
I think this should not have been removed?

Reply:
```python 
    homedir = os.getenv("HOME")
    LD_LIBRARY_PATH = f"{homedir}/.local/lib:{session.env.get('LD_LIBRARY_PATH')}"
    envbindir = session.bin
    session.install("-e", ".[dev]")
```
Added some extra as locally it is breaking with : 
```python 
NameError: name 'LD_LIBRARY_PATH' is not defined
```